tests/OpenTK.Tests/OpenTK.Tests.fsproj

```diff
@@ -73,6 +76,7 @@
     <Reference Include="mscorlib" />
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="System.Drawing" />
```

Comment:
Probably not needed in OpenTK.Tests now that integration tests are their own project.